### ISSUE TITLE
":include" in double quotes doesn't recognized

### ISSUE THREAD

**Issue by zdm** [docsifyjs/docsify#1587]
#### Steps to reproduce

Try to embed external file using for example this link:
```
[readme](https://raw.githubusercontent.com/softvisio/core/main/README.md ":include")
```

#### What is current behaviour

Currently file will not be embedded, because `:include` tag is placed in double quotes.
This is the problem, because `prettier` markdown processor replaces single quotes with double and double quotes are legal for the link title.

#### What is the expected behaviour

I am expecting, that file will be embedded ti the parent document.

#### Other relevant information

- [ x] Bug does still occur when all/other plugins are disabled?

- Your OS: 
- Node.js version: 
- npm/yarn version: 
- Browser version: 
- Docsify version: 4.12.1
- Docsify plugins: 

#### Please create a reproducible sandbox 

[![Edit 307qqv236](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/307qqv236)

**Comment by sy-records** [comment]
you can use

```
[readme](https://raw.githubusercontent.com/softvisio/core/main/README.md "readme :include")
```

**Comment by trusktr** [comment]
Will be fixed by #1598
